### PR TITLE
Milestone 6: Views

### PR DESCRIPTION
**Pull request by anagudelogu**
# In this Milestone I:

- Implemented the design from the sneak peek wireframes.

Blog app views

- Used `after_create` callback to call `update_counters` methods for `postscounter`, `likescounter`, `commentscounter` after their respective creations.
- Updated request specs since expecting a static placeholder text doesn't make any sense now.

I would appreciate any optional comments and feedback about the code in general, this helps me improve my skills and write clean code that follows professional standards. 🤓

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,9 @@
-<h1>here's a list of users</h1>
\ No newline at end of file
+<section class="users">
+  <ul class="users__list">
+    <% @users.each do |user| %>
+    <li>
+      <%= render "shared/user_card", user: user %>
```
Comment (MeqdamAlqudah) [inline app/views/users/index.html.erb]: - [x] Please Clicking on each user should takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @anagudelogu , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:+1:  Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (anagudelogu) [reply app/views/users/index.html.erb]: Hi @MeqdamAlqudah! thank you for your feedback, I just applied it. 🚀

**Review by descholar-ceo — APPROVED** [review]
Hi @anagudelogu 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all changes requested ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**